flink-runtime/src/main/scala/org/apache/flink/runtime/jobmanager/JobManager.scala

```diff
@@ -687,61 +687,23 @@ class JobManager(
     message match {
 
       case RequestAccumulatorResults(jobID) =>
-        try {
```

Comment:
Forwarding a message simply keeps the sender reference. It behaves like a normal message sending otherwise. So of course we need to deal with Exceptions there as well. Thanks!